lib/src/gainmapmath.cpp

```diff
@@ -782,12 +782,14 @@ uint8_t encodeGain(float y_sdr, float y_hdr, uhdr_gainmap_metadata_ext_t* metada
 }
 
 float computeGain(float sdr, float hdr) {
-  if (sdr == 0.0f) return 0.0f;  // for sdr black return no gain
-  if (hdr == 0.0f) {  // for hdr black, return a gain large enough to attenuate the sdr pel
-    float offset = (1.0f / 64);
-    return log2(offset / (offset + sdr));
+  float gain = log2((hdr + 1e-7f) / (sdr + 1e-7f));
```

Comment:
I would recommend making `1e-7f` a global constant since it has been used in multiple places.

Reply:
Done dichen.